examples/evm-multichain-dapp/config/blockchains.json

```diff
@@ -0,0 +1,8778 @@
+{
```

Comment:
This file is huge, I wonder if we can do anything about it? Is this generated/downloaded from somewhere? Maybe we could add instructions on how to download it. What do you think?

Reply:
It is derived from https://chainid.network/chains.json 
Paired it with the key as `chainId`
Like this `chainId:{...BLOCKCHAIN DATA ...}`